hastexo/tasks.py

```diff
@@ -696,18 +697,23 @@ def run_tests(self, ssh):
 
             # Make it executable and run it.
             sftp.chmod(script, 0o775)
-            _, stdout, _ = ssh.exec_command(script)
+            _, stdout, stderr = ssh.exec_command(script)
             retval = stdout.channel.recv_exit_status()
             if retval == 0:
                 score += 1
+            else:
+                error = stderr.read()
```

Comment:
(continued from above) ... or make this `stderr.read.replace('\n', '<br/>')`. Not sure what's the better approach, to be honest.